src/main/java/module/transfer/history/HistoryPane.java

```diff
@@ -217,7 +210,7 @@ public HistoryPane() {
         setLeftComponent(topPanel);
         setRightComponent(playerDetailPanel);
 
-        refresh();
+        //refresh();
```

Comment:
Is that intended?

Reply:
Yes - this refresh is again called from outside and it costs double cpu workload.